openmetadata-ui/src/main/resources/ui/public/locales/en-US/Messaging/Kinesis.md

```diff
@@ -3,77 +3,112 @@
 In this section, we provide guides and references to use the Kinesis connector.
 
 # Requirements
-<!-- to be updated -->
-You can find further information on the Kafka connector in the [docs](https://docs.open-metadata.org/connectors/messaging/kinesis).
+You can find further information on the Kinesis connector in the [docs](https://docs.open-metadata.org/connectors/messaging/kinesis).
 
```

Comment:
Any more details on the type of permission a user would need? I believe the user's IAM policy should have certain access to extract metadata